JsonPath/PathSegment.cs

```diff
@@ -88,7 +88,8 @@ public override string ToString()
 		if (IsShorthand)
 			return $"{((IHaveShorthand)Selectors[0]).ToShorthandString()}";
 		
-		return $"[{string.Join(',', Selectors.Select(x => x.ToString()))}]";
+
```

Comment:
Please remove extra line.  (Single blank, not two.)

Reply:
Resolved